<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tour</class>
 <widget class="QMainWindow" name="tour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tours</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="5,2,4,2" columnminimumwidth="0,0,0,0">
      <item row="3" column="2">
       <widget class="QPushButton" name="reload_table">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Обновить таблицу</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="line_id">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="line_name">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>amount rooms (all)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="line_all_rooms">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>price</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="line_price">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>stars</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="line_stars">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>charter</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="line_charter">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="update_button">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Изменить тур</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="deleteButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить тур по id</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="save_button">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить тур</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3" colspan="3">
       <widget class="QTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">background-color: white;
color: black;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
